### PR TITLE
falco settings: rules header spacing, confirm delete modal added

### PR DESCRIPTION
**Pull request by becky-intelletive**

### CODE REVIEW

File: dash/frontend/src/app/modules/private/pages/falco/falco-org-settings-page/falco-org-settings-page.component.ts [hunk 1]
```diff
@@ -58,18 +59,28 @@ export class FalcoOrgSettingsPageComponent implements OnInit {
...
-          this.alert.danger(err?.error?.metadata || 'Something went wrong.');
-        }
-      });
+    const openAddCluster = this.dialog.open(AlertDialogComponent, {
+      width: '400px',
+      closeOnNavigation: true,
+      disableClose: false,
+      data: {},
+    });
+    openAddCluster.afterClosed().subscribe(shouldDelete => {
```
Comment (kyle-intelletive) [inline dash/frontend/src/app/modules/private/pages/falco/falco-org-settings-page/falco-org-settings-page.component.ts:68]: The after closed subscription should have .pipe(1)

**Review by kyle-intelletive — CHANGES_REQUESTED** [review]
Looks good once the subscription is closed

File: dash/frontend/src/app/modules/private/pages/falco/falco-org-settings-page/falco-org-settings-page.component.ts (hunk 1 above)
Comment (becky-intelletive) [reply dash/frontend/src/app/modules/private/pages/falco/falco-org-settings-page/falco-org-settings-page.component.ts]: Recheck :)

**Review by kyle-intelletive — APPROVED** [review]

File: dash/frontend/src/app/modules/private/pages/falco/falco-org-settings-page/falco-org-settings-page.component.ts (hunk 1 above)
Comment (kyle-intelletive) [reply dash/frontend/src/app/modules/private/pages/falco/falco-org-settings-page/falco-org-settings-page.component.ts]: Nice work

**Pull request merged**